src/libs/ajax/DAR.js

```diff
@@ -1,65 +1,63 @@
 import { fileDownload } from '../../utils/FileDownload'
 import { isNil, mergeAll, omit } from 'lodash/fp'
 import { Config } from '../config'
-import axios from 'axios'
 import { isFileEmpty } from '../utils'
-import { getApiUrl, fetchOk, getOntologyUrl, fetchAny } from '../ajax'
+import { fetchAny, fetchOk, getApiUrl, getOntologyUrl } from '../ajax'
 import { DAAUtils } from '../../utils/DAAUtils'
 import { Metrics } from './Metrics'
 import eventList from '../events'
+import { fetchGet, fetchMultipart, fetchPost, fetchPut } from 'src/libs/ajax/fetchAdapter'
 
 export const DAR = {
   // v2 get for DARs
   getPartialDarRequest: async (darId) => {
     const url = `${await getApiUrl()}/api/dar/v2/${darId}`
-    const res = await fetchOk(url, Config.authOpts())
-    return await res.json()
+    const res = await fetchGet(url, Config.authOpts())
+    return res.data
   },
 
   // v2, v3 Draft DAR Update
   updateDarDraft: async (dar, referenceId) => {
-    // noinspection ES6MissingAwait
-    Metrics.captureEvent(eventList.dar, { action: 'update' })
+    await Metrics.captureEvent(eventList.dar, { action: 'update' })
```

Comment:
Let's not `await` here.  It's preferable to have the Metrics request initiate and process in a separate thread in a non-blocking manner.

```suggestion
    // noinspection ES6MissingAwait
    Metrics.captureEvent(eventList.dar, { action: 'update' })
```